esmvaltool/recipes/recipe_ocean_Landschutzer2014.yml

```diff
@@ -0,0 +1,131 @@
+# ESMValTool
+# Landschutzer2014.yml
+---
+documentation:
+  description: |
+    Recipe to evaluate CO2 Fluxes of marine biogeochemistry models of CMIP5
+    using Landschutzer et al (2.014) dataset.
+    Written by Tomas Lovato, CMCC, tomas.lovato@cmcc.it
+
+  authors:
+    - lova_to
+  
+  maintainer:
+    - lova_to
+
+  references:
+    - acknow_project
+
+  projects:
+    - crescendo
+
+
+datasets:
+# working datasets
+  - {dataset: CanESM2,       project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
+#  - {dataset: HadGEM2-ES,    project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
```

Comment:
Why all the commented out datasets?

Reply:
@bouweandela I will clean up the recipe. Anyway, be aware that You will face the same issue also in other recipes.